src/app/core/subscription-agreement.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SubscriptionAgreementService {
```

Comment:
Can we add a JS doc to this class to explain that this subscription is for the newsletter and not our subscription model?